### DISCUSSION TITLE
Is there any way to improve the gap fill in this app.

### DISCUSSION THREAD

**Discussion by mcornetto** [thygate/stable-diffusion-webui-depthmap-script#468] [Q&A]
I am unable to do any stereoscopic images with depth because the gaps get too large.  The fill is choppy and makes the stereo image look bad.  No fill makes the image look malformed.  Is there any options to install different fills that actually look good?  I know it's possible because other software does it.

**Comment by sonakravtsenko3** [comment]
@mcornetto Hello! Stereoscopic functionality is indeed comparatively primitive. What other applications do you have in mind?

**Comment by mcornetto** [comment]
@semjon00 Immersity is able to do max depth and fills the gaps smoothly.  That's what I meant by other software.

**Comment by sonakravtsenko3** [comment]
Nice to know! This seems like a very interesting project. Sorry, doing something like that would be really difficult, but I hope you could create depthmaps and use them with some other software - then probably the quality will be better.

**Comment by mcornetto** [comment]
@semjon00 yes, I do use this software to create depth maps.  There is nothing that has more of a choice of types of depth maps to make.  BTW, the gap fill doesn't need to be exactly like Immersity, something other than the polyline fills would likely be better. Maybe some kind of blur?

**Comment by sonakravtsenko3** [comment]
That's a nice idea! Would be cool if somebody tried to implement that.
